src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1272,13 +1279,16 @@ static bool addTriggeredSubscriptions_noCache
       long long           lastNotification   = sub.hasField(CSUB_LASTNOTIFICATION)? getIntOrLongFieldAsLongF(sub, CSUB_LASTNOTIFICATION) : -1;
       std::string         renderFormatString = sub.hasField(CSUB_FORMAT)? getStringFieldF(sub, CSUB_FORMAT) : "";
       RenderFormat        renderFormat       = stringToRenderFormat(renderFormatString);
+      ngsiv2::HttpInfo    httpInfo;
+
+      httpInfo.url = sub.hasField(CSUB_REFERENCE)? getStringFieldF(sub, CSUB_REFERENCE) : "";
 
```

Comment:
Probably the "More stuff needed here - the entire HttpInfo must be retreived from DB, obviously" should be converted in a "FIXME PR" comment inline the code, so we don't forget this (easy to forget about the "poor cousin" of nocache functionality ;)